File: lab1/Java/Student.java

public class Student {
	
	private String name;
	private long id;
	private double math, physic, english;
	
	public Student(String name, long id)
	{
		this.name = name;
		this.id = id;
	}
	
	public void setMath(double value)
	{
		math = value;
	}
	
	public void setPhysic(double value)
	{
		physic = value;
	}
	
	public void setEnglish(double value)
	{
		english = value;
	}
	public double getAverage()
	{
		return (math+physic+english) / 3 ;
	}
	public void display()
	{
		System.out.println("Name: " +name);
		System.out.println("ID: " +id);
		System.out.println("Math: " +math);
		System.out.println("Physic: " +physic);
		System.out.println("English: " +english);
		System.out.println("Average: " +getAverage());	
	}
	
	public static void main(String args[])
	{
		Student std1 = new Student("Khai", 1605);
		
		std1.setMath(10);
		std1.setEnglish(9);
		std1.setPhysic(8);
		
		std1.display();
		
	}
	

}

File: lab1/C++/Student.cpp
#include <iostream>

using namespace std;

class student
{
	private:
		string name;
		long id;
		double math, physic, english;

	public:
		student(string name1, long id1)
		{
			name = name1;
			id   = id1;
		}
		double setMath(double value)
		{
			math = value;
		}
		double setPhysic(double value)
		{
			physic = value;
		}
		double setEnglish(double value)
		{
			english = value;
		}
		double getAverage()
		{
			return (math+physic+english) / 3;
		}
		
		void display()
		{
			cout <<"Name: " <<name<<endl;
			cout <<"ID: " <<id<<endl;
			cout <<"Math: " <<math<<endl;
			cout <<"Physic: " <<physic<<endl;
			cout <<"English: " <<english<<endl;
			cout <<"Average: " <<getAverage()<<endl;
		}
		
};

int main() 
{
	student std1 ("Duc", 1605);
	
	std1.setEnglish(10);
	std1.setMath(9);
	std1.setPhysic(8);
	std1.getAverage();
	
	student std2("VInh", 2206);
	std2.setEnglish(3);
	std2.setMath(4);
	std2.setPhysic(5);
	
	std1.display();
	std2.display();
}
